dts/xtensa/espressif/esp32/esp32_common.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 Intel Corporation.
+ * Copyright (c) 2019-2025 Intel Corporation.
```

Comment:
Instead of extending the copyright of the original creator, append a new line adding Espressif's. Do the same for all files that copyright was changed.